### DISCUSSION TITLE
Greats in mania doesnt contribute to 100% accuracy unlike in stable

### DISCUSSION THREAD

**Discussion by enoslayd** [ppy/osu#20543] [Q&A]
twt thread: https://twitter.com/IAmDaru29/status/1575693833605902336

basically, in lazer, greats (300) doesnt lead to a 100% accuracy and requires perfects (300g, rainbow 300s) unlike in stable 

lazer great
![osu_2022-10-02_12-20-33](https://user-images.githubusercontent.com/41866328/193437702-82e61551-8d51-45a7-805c-af73a97787b0.png)

lazer perfect:
![osu_2022-10-02_12-03-36](https://user-images.githubusercontent.com/41866328/193437494-018c9cc2-acf8-4a1a-9217-cbc0f6d1d22d.png)

stable:
![screenshot301](https://user-images.githubusercontent.com/41866328/193437513-93a8d77c-65fc-484e-9f3f-b5392572f643.png)

**Comment by peppy** [comment]
I think this might be misworded.

They *are* contributing as your screenshot shows. What you are proposing is that *perfects* do not contribute *differently* to greats. As in making them redundant for accuracy again. Correct?

**Reply by enoslayd** [reply]
I honestly don't mind the current implementation but i don't know about others

**Reply by peppy** [reply]
The change *was* intentional to gauge if it's a direction we can go.

**Comment by smoogipoo** [comment]
Don't mania players generally care about ratio of perfects/greats (maxs/300s)? I think it makes sense as it is.

The obvious downside is SSs are very hard to obtain, but we haven't figured out the kinks there yet.
